### DISCUSSION TITLE
Understanding and plotting recharge

### DISCUSSION THREAD

**Discussion by lauramenezess** [pastas/pastas#797] [Q&A] [answered]
Hello,

I am trying to understand why my plot for recharge doesn't make sense when using a linear recharge model.

I have the following calibrated and validated model using a linear recharge model:
![Figure_1_Simul](https://github.com/user-attachments/assets/1430568f-84ec-420a-b863-804c64d500ae)
![Figure_2_Validation](https://github.com/user-attachments/assets/4d2fd295-da62-43be-95e5-a0e4b9d068cc)
For the plot I have this:
![Recharge_Secteur2](https://github.com/user-attachments/assets/0dd955e2-f314-40ca-b54c-eb437367f7c9)

For comparison I also have a pastas model for the same piezometer using a series of recharge estimated in another model (GWN-BW), which I add in pastas as settings="prec". I have fixed the well parameters to the same values of the model above, so I have this:
![Figure_14_Simul_GWN](https://github.com/user-attachments/assets/413221cc-9645-4460-8ef7-5819b183954a)
![Figure_14_Valid_GWN](https://github.com/user-attachments/assets/2f123fbb-e9f3-42fc-b5ce-a335fe627e2c)
And the recharge plot:
![Recharge_Secteur2_GWN](https://github.com/user-attachments/assets/4e449418-d27e-4ae4-84a6-13517993d1b9)

Do you have any insights on why the plot for the recharge when using the linear model gives us negative values? Is it the "f" parameter (0.7) overestimating the ETP?

We are trying to continue with the linear recharge model because it's way faster and simple for the purpose of water table predictions than running the GWN model each time to update the recharge. But we also want to use this pastas model for characterization and comparison between zones.
Should we consider the modeled recharge using GWN as a guide to calibrate our linear recharge model?

A bit long and maybe vague questions but any advice or comment from you would be helpful :)

**Comment by raoulcollenteur** [comment, marked as answer]
Hi @lauramenezess,

The linear model does not really estimate 'recharge', but precipitation excess. So the flux computed with the linear model should not be interpreted as recharge unless for very specific hydrogeological settings. Negative values are because the linear recharge model is not water limited and thus always evaporates up to f*pot. evaporation. Negative values would mean that groundwater is evaporated. So only if that applies to your system can you try and interpret it that way.

For water table predictions getting the recharge right might not be important though, so you can simply not look at it. If you are interested in recharge, you probably want to use a nonlinear model.

Hope this helps!

Cheers,
Raoul

**Comment by dbrakenhoff** [comment]
If you hadn't found it already, it's also worth checking out Raoul's article on estimating recharge from time series models: https://doi.org/10.5194/hess-25-2931-2021

**Comment by lauramenezess** [comment]
Ok, great. Thanks @raoulcollenteur and @dbrakenhoff !
